### PR TITLE
WRN-18221: Close color pickers on touch outside or on ESC key.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Close color pickers on touch outside or on ESC key.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed the base component from ButtonBase wrapped with ContextualPopupDecorator to Popup.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-18221

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: Drawing/ColorPicker.js [hunk 1]
```diff
@@ -240,24 +218,30 @@ const ColorPickerBase = kind({
...
 				/>
+				<Popup
+					onClose={handleClosePopup}
+					open={popupOpen}
+					position="left"
+					scrimType="transparent"
+				>
+					<Row>
+						<BodyText>{text}</BodyText>
+						<Button className={css.closeButton} icon={'x'} onClick={handleClosePopup} />
```
Comment (stanca-pop-lgp) [inline Drawing/ColorPicker.js:241]: You can replace icon 'x' with sandstone icon code 'closex'.

Comment (alexandrumorariu) [reply Drawing/ColorPicker.js]: Done

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**